serverless-plugin/README.md

```diff
@@ -6,8 +6,8 @@
 SLIC Watch provides a CloudWatch Dashboard and Alarms for:
```

Comment:
I don't know id this is what you were implying earlier, but I am thinking that we might have a build step in our pipeline where this readme is copied over before publish. This way we don't have to checkout a clone of the original file into git...

Reply:
That's be great. We would need to update relative links.
An alternative is to have a parent readme that links to the plugin one. I'm okay with either idea.